.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -181,21 +183,23 @@ task TransformGISAID {
     mv /ncov-ingest/source-data/czgenepi_locations.tsv \
        /ncov-ingest/source-data/gisaid_geoLocationRules.tsv
 
-    # decompress the gisaid dataset and transform it.
-    ${aws} s3 cp --no-progress "s3://${raw_gisaid_s3_bucket}/${raw_gisaid_s3_key}" - | zstdmt -d > gisaid.ndjson
-    /ncov-ingest/bin/transform-gisaid     \
-        gisaid.ndjson                     \
+    # decompress the raw dataset and transform it.
+    ${aws} s3 cp --no-progress "s3://${raw_data_s3_bucket}/${raw_data_s3_key}" - | zstdmt -d > genbank.ndjson
+    /ncov-ingest/bin/transform-genbank    \
+        genbank.ndjson                    \
         --output-metadata metadata.tsv    \
         --output-fasta sequences.fasta    \
+        --biosample=""                    \
         --output-unix-newline
 
     zstdmt sequences.fasta
+    xz metadata.tsv
```

Comment:
Just curious: why do we use `zstdmt` for compression on the first, but then `xz` for the other?

Reply:
I'm just trying to keep to the conventions of the previous script, even though switching between 3 different compression formats is super confusing 🤯